app/lib/router/services.js

```diff
@@ -26,6 +26,30 @@ Router.route('/projects/:id/hosts/:hid/services/:sid', {
   }
 })
 
+
+Router.route('/projects/:id/hosts/:hid/services/:sid/:page/next', {
+  name: 'nextService',
+  controller: 'ProjectController',
+  onBeforeAction: function () {
+    console.log("Executing service next route")
```

Comment:
Yes, forgot to remove that debugging. Will upload a new commit tonight.

On Tue, Jan 26, 2016 at 5:00 AM, Tom Steele <notifications@github.com>
wrote:

> In app/lib/router/services.js
> <https://github.com/lair-framework/lair/pull/81#discussion_r50814670>:
>
> > @@ -26,6 +26,30 @@ Router.route('/projects/:id/hosts/:hid/services/:sid', {
> >    }
> >  })
> >
> > +
> > +Router.route('/projects/:id/hosts/:hid/services/:sid/:page/next', {
> > +  name: 'nextService',
> > +  controller: 'ProjectController',
> > +  onBeforeAction: function () {
> > +    console.log("Executing service next route")
>
> Can we remove console.log here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lair-framework/lair/pull/81/files#r50814670>.
>